tests/languages/clojure/boolean_feature.test

```diff
@@ -0,0 +1,13 @@
+false
+true
+
```

Comment:
Could you add a test for `nil` here? Since it's handled in the `boolean` pattern.

Reply:
Resolved